CHANGELOG.md

```diff
@@ -1,7 +1,13 @@
 # Change Log - @itwin/insights-client
 
+## 0.11.0
```

Comment:
There is no need to bump the prerelease minor version on every change. You are adding features by exposing things and that is not a breaking change.